DXMainClient/DXGUI/Multiplayer/CnCNet/PrivateMessagingWindow.cs

```diff
@@ -18,6 +18,12 @@ namespace DTAClient.DXGUI.Multiplayer.CnCNet
 {
     internal class PrivateMessagingWindow : XNAWindow, ISwitchable
     {
+        // these are used by the "invite to game" feature in the
+        // context menu and are kept up-to-date by the lobby
+        public string inviteChannelName;
+        public string inviteGameName;
+        public string inviteChannelPassword;
```

Comment:
Public fields and properties start with a capital letter. Also, turn them into properties with getters and setters (for example `public string InviteChannelName { get; set; }`. Also see the note related to these that I left for the CnCNet lobby.

Reply:
Have made these private and permitted access via set/clear methods only as this seems to address both comments pretty cleanly.